packages/containers/src/List/List.connect.js

```diff
@@ -4,42 +4,43 @@ import { cmfConnect } from '@talend/react-cmf';
 import Container, { DEFAULT_STATE } from './List.container';
 import {
 	configureGetFilteredItems,
-	configureGetPagination,
 	configureGetPagedItems,
+	configureGetPagination,
+	getCollectionItems,
 } from './selector';
 
 function componentId(ownProps) {
 	return ownProps.collectionId;
 }
 
-function getItems(state, config) {
-	const items = configureGetFilteredItems(config)(state);
-
-	return items || new List();
-}
-
-function getPagedItems(state, config) {
-	const items = configureGetPagedItems(config)(state);
+function getPagedItems(state, config, filteredItems) {
+	const items = configureGetPagedItems(config, filteredItems)(state);
 
 	return items || new List();
 }
 
 export function mapStateToProps(state, ownProps, cmfProps) {
 	const props = {};
-	const config = {
+	let filteredItems;
+	const collectionItems = getCollectionItems(state, ownProps.collectionId);
+
+	props.config = {
```

Comment:
You attach the config to props but you don't use it except in this function.
Please remove it, this will introduce perf issues, as it's always a new object at each render.